packages/console/src/pages/Roles/components/CreateRoleForm/index.tsx

```diff
@@ -55,21 +64,63 @@ function CreateRoleForm({ onClose }: Props) {
     })
   );
 
+  const isRolesOverQuota = isOverQuota({
+    quotaKey: 'rolesLimit',
+    plan: currentPlan,
+    usage: totalRoleCount,
+  });
+
+  const isScopesPerRoleOverQuota = isOverQuota({
+    quotaKey: 'scopesPerRoleLimit',
+    plan: currentPlan,
+    usage: roleScopes.length - 1,
```

Comment:
Could you add a comment on why minus 1 here?

Reply:
Added, and rename the `isOverQuota` to `hasReachedQuotaLimit`, since we use `usage >= limit` there.